recipe/meta.yaml

```diff
@@ -1,23 +1,30 @@
 {% set version = "3.28.1" %}
+{% set version_major_minor = ".".join(version.split(".")[:2]) %}
 
 package:
   name: cmake-no-system
   version: {{ version }}
 
 source:
-  url: https://gitlab.kitware.com/cmake/cmake/-/archive/v{{ version }}/cmake-v{{ version }}.tar.bz2
-  sha256: 7474902936e07f147feb8d42d59cca5c2dc60aa8ab3331ffc10cf58f3f0061d4
+  - url: https://gitlab.kitware.com/cmake/cmake/-/archive/v{{ version }}/cmake-v{{ version }}.tar.bz2
+    sha256: 7474902936e07f147feb8d42d59cca5c2dc60aa8ab3331ffc10cf58f3f0061d4
+  - url: https://cmake.org/files/v{{ version_major_minor }}/cmake-{{ version }}-windows-x86_64.zip  # [win]
+    sha256: 671332249bc7cc7424523d6c2b5edd3e3de90a43b8b82e8782f42da4fe4c562d                        # [win]
+    folder: cmake-{{ version }}-windows                                                             # [win]
 
 build:
   number: 0
-  skip: true  # [win]
+  ignore_run_exports:
+    - vc
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - make
-    - patch
+    - make  # [unix]
+    - ninja
+  run:
+    - vs2015_runtime  # [win]
```

Comment:
truthfully i don't really understand this line.

Reply:
Why did you add it then? Shouldn't this come through the compiler run_exports?

Reply:
i took most of the work from jschueller and rebased it. 

Correct, I believe it should come from run_export. let me try.